### DISCUSSION TITLE
Cannot access'store' before initialization Uncaught

### DISCUSSION THREAD

**Discussion by hkllin** [vuejs/pinia#2332] [Help and Questions]
<img width="431" alt="image" src="https://github.com/vuejs/pinia/assets/93277196/e99312c1-e424-4108-ac39-49bf771e4c4b">

**Comment by hkllin** [comment]
![Uploading image.png…]()

**Comment by hkllin** [comment]
<img width="677" alt="image" src="https://github.com/vuejs/pinia/assets/93277196/83820b9a-348a-476f-9ba7-f05f404b14d2">
我认识是pinia还没有初始化就进行了 创建i18n

**Reply by posva** [reply]
You need to pass the pinia instance https://pinia.vuejs.org/core-concepts/outside-component-usage.html

**Comment by hkllin** [comment]
useAppStoreWithOut 这个方法报错了
![Uploading image.png…]()

**Comment by hkllin** [comment]
![image](https://github.com/vuejs/pinia/assets/93277196/811663f9-e3e6-4fdd-b87a-987c99c6043f)
 经过这样的骚操作解决了，你们有什么办法吗
